docs/product-getting-started-guide/src/main/asciidoc/task_new-project.adoc

```diff
@@ -1,32 +1,25 @@
 [id='_importing_a_getting_started_business_project']
```

Comment:
You'll need to update all file names and anchor IDs in this guide per the [file and anchor format](http://ccs-jenkins.gsslab.brq.redhat.com:8080/view/Topic%20branches/job/doc-Red_Hat_JBoss_BxMS_Contribution_Guide-branch-shared-repo/lastSuccessfulBuild/artifact/index.html#_file_names_and_anchor_names) we're using going forward. Dashes, not underscores, and nothing like that preceding underscore. And follows the naming format described in the linked contrib guide section.

Before you change any anchor ID, however, remember to first do a quick whole-project search for it throughout all doc folders in Asciidoc (Find > Find in project), and update any cross-references that link to it so that they use the new name instead. Otherwise you'll have broken links. Then do the same thing for the file name: Search throughout to see if existing includes point to that file, and update as needed. Then you can make the change to the anchor ID and file name. You do that every time you change one. Doesn't take too long, but is critical.

In order for this to work, you must be at the "docs" folder level in asciidoc, i.e., in kie-docs/docs. If you're further down, then when you do a whole-project search, it only searches in whichever folder/level you're in, and you'll not be high enough up.

Reply:
Done. Was holding off, but better now than later. Thank you for the reminder.